### PR TITLE
[Changes made] Add MySQL for Developers course

### PR DESCRIPTION
**Pull request by hollylawly**
## What does this PR do?
Add resource

### Description
Added a new [MySQL for Developers course](https://planetscale.com/learn/courses/mysql-for-developers/introduction/course-introduction) under English courses.

### Why is this valuable (or not)?
It's a 54 video course that teaches engineers how to improve their MySQL skills. It goes very in depth with useful and practical ways to improve queries, schemas, indexes, and provides real-world examples.

### How do we know it's really free?
Email is required, but you can put in a fake address to test it.

### For book lists, is it a book? For course lists, is it a course? etc.
It's a course.

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -567,6 +567,7 @@
 * [Learn SQL](https://popsql.com/learn-sql) - PopSQL
 * [Learn SQL](https://www.sololearn.com/learning/1060) - *registration required*
 * [Learn SQL: SQL Tutorial for Beginners](https://www.programiz.com/sql) - Programiz
+* [MySQL for Developers](https://planetscale.com/learn/courses/mysql-for-developers/introduction/course-introduction) - Aaron Francis (PlanetScale) _(email address required)_
```
Comment (eshellman) [inline courses/free-courses-en.md:570]:
```suggestion
* [MySQL for Developers](https://planetscale.com/learn/courses/mysql-for-developers/introduction/course-introduction) - Aaron Francis (PlanetScale)
```
seems like you can access the content without registering

Comment (hollylawly) [reply courses/free-courses-en.md:570]: It does gate it after a couple of videos so just wanted to mention it. I'm good to remove it though if it's unnecessary.

**Comment by eshellman** [comment]
we're more tolerant of that in the courses lists. thanks for checking

File: courses/free-courses-en.md (hunk 1 above)
Comment (Thenlie) [reply courses/free-courses-en.md:570]: +1 for removing this. Everything else looks good.

Once you have made the requested changes, please update the PR title to include “(changes made)” and a reviewer will take another look as soon as they are available. Thanks!

Comment (hollylawly) [reply courses/free-courses-en.md]: Just updated! Thank you!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**